### ISSUE TITLE
Can't detect release_group "x0r"

### ISSUE THREAD

**Issue by Apollo2000** [guessit-io/guessit#551]
File name: Shaolin 2011.720p.BluRay.x264-x0r.mkv
Array
(
    [title] => Shaolin
    [year] => 2011
    [screen_size] => 720p
    [source] => Blu-ray
    [video_codec] => H.264
    [bonus] => 0
    [bonus_title] => r
    [container] => mkv
    [mimetype] => video/x-matroska
    [type] => movie
)

All files with group "x0r" isn't detected.

GuessIt 3.0.0,  Rebulk 0.9.0

Thank you.

**Comment by Apollo2000** [comment]
+-------------------------------------------------------+
+                   GuessIt 3.0.0                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+

**Comment by duramato** [comment]
```
guessit "Shaolin 2011.720p.BluRay.x264-x0r.mkv" --expected-group "x0r"
For: Shaolin 2011.720p.BluRay.x264-x0r.mkv
GuessIt found: {
    "title": "Shaolin",
    "year": 2011,
    "screen_size": "720p",
    "source": "Blu-ray",
    "video_codec": "H.264",
    "release_group": "x0r",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "movie"
}
```

**Comment by Toilal** [comment]
Even if it can be workarounded with expectedGroup option, there's a problem with `bonus`/`bonus_title` matcher. It should not match when there's no separator between `bonus` and `bonus_title`

**Comment by ratoaq2** [comment]
Fixed by 4c10925e97046b8801f4d07800fae5cd6500dc37
